packages/app-store/hubspotothercalendar/components/InstallAppButton.tsx

```diff
@@ -0,0 +1,18 @@
+import type { InstallAppButtonProps } from "@calcom/app-store/types";
+
+import useAddAppMutation from "../../_utils/useAddAppMutation";
+
+export default function InstallAppButton(props: InstallAppButtonProps) {
+  const mutation = useAddAppMutation("hubspot_other_calendar");
+
+  return (
+    <>
+      {props.render({
+        onClick() {
+          mutation.mutate("");
+        },
+        loading: mutation.isLoading,
+      })}
```

Comment:
Hey maybe we could leverage installs to pass data to jitsu to have stats about installs/usage/continued usage cohorts through time of each app to know which ones to further improve/develop

Reply:
Also a very good idea. We can tackle it in a separate PR for all apps so we are covered.

Reply:
We can do this once in the base Install button